apps/web-app/pages/[orgShortcode]/convo/new.vue

```diff
@@ -726,9 +726,10 @@
         :schema="z.string().trim().min(1).max(128)" />
     </div>
     <span class="text-sm font-medium">Message</span>
+
     <UnEditor
       v-model:modelValue="messageEditorData"
-      class="min-h-[150px] overflow-hidden" />
+      class="min-h-full overflow-hidden" />
```

Comment:
did you test this?
we dont want a full height editor if theres no messages

Reply:
It works full height in new message only, in reply UI it is min height
I have tested it already, 150px was not covering the whole Message Box